addons/project_timesheet_holidays/tests/test_timesheet_holidays.py

```diff
@@ -95,54 +95,48 @@ def setUp(self):
 
     def test_validate_with_timesheet(self):
         # employee creates a leave request
-        number_of_days = (self.leave_end_datetime - self.leave_start_datetime).days
         holiday = self.Requests.with_user(self.user_employee).create({
             'name': 'Time Off 1',
             'employee_id': self.empl_employee.id,
             'holiday_status_id': self.hr_leave_type_with_ts.id,
-            'date_from': self.leave_start_datetime,
-            'date_to': self.leave_end_datetime,
-            'number_of_days': number_of_days,
+            'request_date_from': self.leave_start_datetime,
+            'request_date_to': self.leave_end_datetime,
```

Comment:
Why not use dates instead instead of datetimes?

Reply:
I could, but in the end it doesn't really matter, datetimes are truncated to dates when passed to date fields in odoo.